crates/krator/src/lib.rs

```diff
@@ -11,3 +11,4 @@ pub mod state;
 pub use object::{ObjectState, ObjectStatus};
 pub use operator::Operator;
 pub use runtime::OperatorRuntime;
+pub use state::{State, Transition, TransitionTo};
```

Comment:
Should we put most of these (including the ones above) in a `prelude`?

Reply:
I started doing this actually. It got lost in the rebase. I will add later as the API stabilizes.